src/pages/ProductPage/ProdDetail/ProdDetail.jsx

```diff
@@ -0,0 +1,23 @@
+import "./ProdDetail.scss";
+
+export default function ProdDetail({
+  company,
+  model,
+  description,
+  originalPrice,
+  price,
+  discount,
+}) {
+  return (
+    <div className="prodDetail">
+      <h2>{company}</h2>

```

Comment:
Let's use CSS class to style it:

```suggestion
      <h2 className="product-detail__title">{company}</h2>
```